docs/source/example_use.rst

```diff
@@ -1,3 +1,343 @@
-Example Use Case
-================
+###############################
+Example Use Case: The News Room
+###############################
 
+Below we'll walk you through an example of how Orchestra could be used in a
+newsroom by journalists, editors, and photographers to craft a story. The code
+for this example can be found `in our github repo
+<https://github.com/unlimitedlabs/orchestra/tree/stable/journalism_workflow>`_.
```

Comment:
This will exist once we move the stable tag (verify at https://github.com/unlimitedlabs/orchestra/tree/journalism_workflow_doc/journalism_workflow)